docs/en/tools-and-utilities/static-files-disk-uploader.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: /en/operations/utilities/static-files-disk-uploader
+title: clickhouse-static-files-disk-uploader
+keywords: [clickhouse-static-files-disk-uploader, utility, disk, uploader]
+---
+
+# clickhouse-static-files-disk-uploader
+
+Outputs a data directory containing metadata for a specified ClickHouse table. This metadata can be used to create a ClickHouse table on a different server containing a read-only dataset backed by a `web` disk.
+
+Do not use this tool for migrating data. Instead use the [`BACKUP` and `RESTORE` commands](/docs/en/operations/backup).
+
+## Usage
+
+```
+$ clickhouse static-files-disk-uploader [args]
+```
+
+## Commands
+
+|Command|Description|
+|---|---|
+|`-h`, `--help`|Prints help information|
+|`--metadata-path [path]`|The path containing metadata for the specified table|
+|`--test-mode`|Enables `test` mode which submits a PUT request to the given URL with the table metadata|
+|`--link`|Creates symlinks instead of copying files to the output directory|
+|`--url [url]`|Web server URL for `test` mode|
+|`--output-dir [dir]`|Directory to output files in `non-test` mode|
+
+## Retrieve metadata path for the specified table
+
+When using `clickhouse-static-files-disk-uploader`, you will need to obtain the metadata path for your desired table.
+
+1. Run the following query specifying your target table and database:
+
+<br />
```

Comment:
Are these `<br />` elements needed? If we want more padding between list elements and codeblocks, we should make changes to the CSS rather than hardcoding breaks like this.

Reply:
We should–just for now since I wanted to commit this quickly the `<br/>` is temporarily to make it formatted nicely.